server/scripts/database-config-utils.ts

```diff
@@ -21,17 +21,18 @@ export function getEnvVars() {
   if (fs.existsSync(envVarsFilePath)) {
     data = { ...data, ...dotenv.parse(fs.readFileSync(envVarsFilePath)) };
   }
+
   data = {
     ...data,
     ...(data.DATABASE_URL && buildDbConfigFromDatabaseURL(data)),
+    ...(data.TOOLJET_DB_URL && buildDbConfigFromDatabaseURL(data))
```

Comment:
we can combine the conditional check here:

`...((data.DATABASE_URL || data.TOOLJET_DB_URL) && buildDbConfigFromDatabaseURL(data))`